src/d3a/d3a_core/export.py

```diff
@@ -122,10 +122,13 @@ def delete_exported_files(self):
             os.remove(zip_file_with_ext)
         shutil.rmtree(str(self.directory))
 
-    def export(self, export_plots=True):
+    def export(self, export_plots=True, power_flow=None):
```

Comment:
Why not using False as default value here?

Reply:
I see now. It snot a flag. But why parsing as a parameter and not saving it as a member of the simulation class like all other stats as well?

Reply:
Im not sure if I have exactly understood, `power_flow` is an instance variable that belongs to `Simulation` class and `export` is also an instance variable of `Simulation` class. So, I had to pass the instance variable `self.power_flow` of `Simulation` class as a parameter to a method that belongs to other instance. We can also discuss for better clarification.

Reply:
Yes, you are right. Didnt understand it before. Thanks!